js/activity.js

```diff
@@ -4652,6 +4658,9 @@ function Activity() {
         // Put the boundary in the blocks container so it scrolls
         // with the blocks.
 
+        rhythm = new RhythmRuler();
```

Comment:
Why are you creating rhythm ruler here?

Reply:
@walterbender  I have done this to make a object and using this I am accessing **Rulers of** 
rhythm maker .

Reply:
The rhythm ruler you define here is not seen by widget, so it will not have any of the widget changes we want to save.

I thought I had explained earlier that you can access the Rhythm Ruler in logo.js the same way it is accessed in WidgetBlocks.js

Look at the code here:
https://github.com/sugarlabs/musicblocks/blob/master/js/blocks/WidgetBlocks.js#L864